Vagrantfile

```diff
@@ -36,6 +36,10 @@ Vagrant.configure("2") do |config|
       v.name = "jenkins"
     end
 
+    node.vm.provider "vmware_fusion" do |v|
+      v.vmx["memsize"] = "1024"
+    end
+
```

Comment:
Yes... it will just not set the mem size to 1024.

On 9 February 2015 at 12:03, Matthew Mead-Briggs <notifications@github.com>
wrote:

> In Vagrantfile
> <https://github.com/ministryofjustice/jenkins-formula/pull/9#discussion_r24323692>
> :
>
> > @@ -36,6 +36,10 @@ Vagrant.configure("2") do |config|
> >        v.name = "jenkins"
> >      end
> >
> > +    node.vm.provider "vmware_fusion" do |v|
> > +      v.vmx["memsize"] = "1024"
> > +    end
> > +
>
> Will this still run if I don't have fusion?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/jenkins-formula/pull/9/files#r24323692>
> .
>